### ISSUE TITLE
Twitch clips wont play while extension enabled Chrome

### ISSUE THREAD

**Issue by crashbash2020** [ejci/Chrome-Audio-EQ#90]
Clipped twitch videos cant play for some reason (no buffering or audio/video). Regular streaming video and audio on twitch works fine

disabling extension and refresh fixes issue

Chrome 66 on windows 8.1

**Comment by TWlTCH** [comment]
Confirmed. Still an issue. BUMP

**Comment by NTripleOne** [comment]
Yes, for some reason the extension is causing bogus "No 'Access-Control-Allow-Origin' header is present on the requested resource" errors on some videos, including clips on twitch.

**Comment by haitua** [comment]
I too got those bugs on twitch clips, and also instagrams story/videos, embed videos

**Comment by xodotLBO** [comment]
Last update for the extension was like 5-8 years ago. Doubt it'll get any fixes.
